package.json

```diff
@@ -24,13 +24,13 @@
   ],
   "license": "Apache-2.0",
   "dependencies": {
-    "axios": "^0.21.4",
-    "log4js": "^0.6.14",
-    "moment": "^2.22.2",
-    "uuid": "^3.0.0"
+    "axios": "^0.24.0",
+    "log4js": "^6.3.0",
```

Comment:
```suggestion
    "log4js": "^6.4.0",
```

Reply:
To resolve a security vulnerability with log4js:
https://www.whitesourcesoftware.com/vulnerability-database/CVE-2022-21704